plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/AdvancedConnectionEditor.java

```diff
@@ -126,7 +126,7 @@ public void widgetSelected(SelectionEvent e) {
 					}
 				});
 				GridDataFactory.fillDefaults()
-				.align(SWT.FILL, SWT.CENTER).grab(true, false).span(1, 1)
+				.align(SWT.FILL, SWT.CENTER).hint(100, SWT.DEFAULT)
```

Comment:
Why not using org.jboss.tools.openshift.internal.common.ui.utils.UIUtils.setDefaultButtonWidth ?

Reply:
UIUtils.setDefaultButtonWidth is as effective while being simpler with the same effect.